{
  assertThrows(future,exceptionClass,null,null);
}
